velox/functions/remote/server/CMakeLists.txt

```diff
@@ -24,3 +24,18 @@ add_executable(velox_functions_remote_server_main RemoteFunctionServiceMain.cpp)
 target_link_libraries(
   velox_functions_remote_server_main velox_functions_remote_server
   velox_functions_prestosql)
+
+add_library(velox_functions_remote_server_rest RemoteFunctionRestService.cpp)
```

Comment:
Ok, so the idea is that a user can implement Velox functions and add them to this REST server and serve them up for execution though a remote velox engine?